css/styles.css

```diff
@@ -0,0 +1,170 @@
+*{
+	box-sizing: border-box;
+}
+
+h1{
+	color: yellow;
+  font-family: 'Merriweather', sans-serif;
+  float: left;
+  margin-left: 30px;
+}
+
+a{
+	color: white;
+  margin-right: 5%;
+  text-decoration: none;
+}
+
+#yellowbox{
+    background-color: yellow;
+    color: black;
+    padding: 10px 35px;
+    text-align: center;
+    display: inline-block;
+    font-size: 16px;
+    font-family: 'Oswald', sans-serif;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
+    position: relative;
+    top: 20px;
+    left: 15px;
+    right: 25px;
+    bottom: 20px;
+}
+
+button{
+    background-color: lightgray;
+    color: black;
+    padding: 10px 30px;
+    text-align: center;
+    display: inline-block;
+    font-size: 16px;
+    font-family: 'Oswald', sans-serif;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
+}
+
+.container{
+  width: 850px;
+  margin: 0 auto;
+  overflow: auto;
+  height: 980px;
+}
+
+.content {
+  width: 20%;
+  height: 50%;
+  position: absolute;
+  left: 50%;
```

Comment:
@kelseycoughlin adjust the position of this component so that it closer to the given design 🔨